pyccl/halos/profiles.py

```diff
@@ -166,19 +171,21 @@ def real(self, cosmo, r, M, a, mass_def=None):
             are scalars, the corresponding dimension will be
             squeezed out on output.
         """
-        if getattr(self, '_real', None):
+        if getattr(self, "_real", None):
             f_r = self._real(cosmo, r, M, a, mass_def)
-        elif getattr(self, '_fourier', None):
-            f_r = self._fftlog_wrap(cosmo, r, M, a, mass_def,
-                                    fourier_out=False)
+        elif getattr(self, "_fourier", None):
+            f_r = self._fftlog_wrap(cosmo, r, M, a,
+                                    mass_def, fourier_out=False)
         else:
-            raise NotImplementedError("Profiles must have at least "
-                                      " either a _real or a "
-                                      " _fourier method.")
+            raise NotImplementedError(
```

Comment:
Same here, regarding the closing bracket at the end, and the 4-spaces padding of the arguments. This is one way to do it, and the other one is aligning all the function arguments right after the function opening bracket. These are both OK according to `flake8`. I'd personally leave it as it was for consistency with the rest of the `pyccl` code, where the second rule has been used.